AWSSDKSwiftCLI/Sources/AWSCLIUtils/Process+Utils.swift

```diff
@@ -48,30 +48,31 @@ func _run(_ process: Process) throws {
     try ProcessRunner.standard.run(process)
 }
 
-func _runReturningStdOut(_ process: Process) throws -> String? {
+public func _runReturningStdOut(_ process: Process) throws -> String? {
     let stdOut = Pipe()
     process.standardOutput = stdOut
     
-    var data = Data()
-    stdOut.fileHandleForReading.readabilityHandler = { handle in
-        data += handle.availableData
-    }
-    
     try _run(process)
     process.waitUntilExit()
     
+    let data = try stdOut.fileHandleForReading.readToEnd() ?? Data()
```

Comment:
The `readToEnd()` method is used instead of the `readabilityHandler` above because sometimes the data was being returned before all of it had been written.  `readToEnd()` guarantees that the file handle closes & all the data is read before it returns.